keyboards/bop/config.h

```diff
@@ -0,0 +1,39 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x666b // fk
+#define PRODUCT_ID      0x626F // bo
+#define DEVICE_VER      0x0001 // rev 1
+#define MANUFACTURER    fruitykeeb
+#define PRODUCT         bop
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 20
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { B7, D0, D1, D2, D3, D4 }
+#define MATRIX_COL_PINS { D5, C5, B0, B1, B2, B3, B4, B5, B6, E7, E6, F0, F7, F6, F5, F4, F3, F2, F1, C6 }
+//    If your board is spamming the end column, change C7 to C6 in the line above and short those pins on the controller
+                        
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCe 5
```

Comment:
Case matters for defines. 
```suggestion
#define DEBOUNCE 5
```